plugins/outputs/wavefront/wavefront.go

```diff
@@ -77,30 +78,50 @@ func (*Wavefront) SampleConfig() string {
 	return sampleConfig
 }
 
+func SenderURLFromURLAndToken(rawURL, token string) (string, error) {
+	newURL, err := url.Parse(rawURL)
+	if err != nil {
+		return "", fmt.Errorf("could not parse the provided Url: %s", rawURL)
+	}
+	newURL.User = url.User(token)
+
+	return newURL.String(), nil
+}
+
+func SenderURLFromHostAndPort(host string, port int) string {
+	newURL := url.URL{
+		Scheme: "tcp",
+		Host:   fmt.Sprintf("%s:%d", host, port),
+	}
+	return newURL.String()
+}
```

Comment:
Do these new functions need to be public?

Reply:
That's a fair question. They might make sense to be private. I made them public because they have tests, but I don't suppose there are really any situations where someone outside the package or its tests would call them.

Reply:
Ah right, thanks!